scripts/init_database.py

```
@@ -0,0 +1,170 @@
+        logger.error(f"❌ 테이블 생성 실패: {e}")
+        return False
+
+
+async def create_indexes():
+    """성능 향상을 위한 인덱스를 생성합니다."""
+    logger.info("🔍 인덱스 생성 시작...")
+    
+    indexes = [
+        # ChatSession 인덱스
+        "CREATE INDEX IF NOT EXISTS idx_chat_session_user_id ON chat_sessions(user_id);",
+        "CREATE INDEX IF NOT EXISTS idx_chat_session_active ON chat_sessions(is_active) WHERE is_active = true;",
```

Pylint (3.3.7)

[convention] Line too long (113/100)

(C0301)